component-catalog-connectors/airflow-example-components-connector/README.md

```diff
@@ -0,0 +1,60 @@
+## Elyra pipeline component examples catalog
+
+This catalog connector provides access to example pipeline components for Apache Airflow.
+
+### Install the component examples
+
+You can install the component examples from PyPI or source code. Note that a **rebuild of JupyterLab is not required**.
+
+**Prerequisites**
+
+- [Install Elyra](https://elyra.readthedocs.io/en/stable/getting_started/installation.html) (version 3.3 and above).
```

Comment:
Won't this include a dependency on the examples and prevent the need for the `pip install` below?

Hmm - or are we not doing that since this is now split by runtime-type?

Reply:
My main motivation was to explicitly highlight that version 3.3 is required and ...

You are right that

```
pip install elyra[all]
```
(pending PR) will install the examples (for KFP and AA). What is not clear is if users actually install that way or omit `[all]`. In the latter case, an explicit install would still be required. In a way I am trying to cover all bases without if then else (:

Once we have runtime specific targets (e.g. `elyra[kfp-only]`) only the runtime specific examples will be installed.

Reply:
> Once we have runtime specific targets (e.g. elyra[kfp-only]) only the runtime specific examples will be installed.

I personally think we're going to need to move to a mono-repo that supports elyra, elyra-kfp, and elyra-airflow packages since I don't think the `extra_requires` approach will handle entry_points in the manner we need.